src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.html

```diff
@@ -74,16 +85,23 @@
               ng-change="changeValueAxis($index)"
             >
               <option ng-repeat="axis in editorState.params.valueAxes track by axis.id" value="{{axis.id}}">{{axis.name}}</option>
-              <option value="new">New Axis ...</option>
+              <option
+                value="new"
+                i18n-id="kbnVislibVisTypes.controls.pointSeries.series.newAxisLabel"
+                i18n-default-message="New Axis ..."
```

Comment:
Since we're anyway touching that string now, could we please use the correct ellipsis character here, instead of three dots.